kratos/solving_strategies/builder_and_solvers/builder_and_solver.h

```diff
@@ -521,50 +624,54 @@ class BuilderAndSolver
     }
 
     /**
-     * This function is designed to be called once to perform all the checks needed
+     * @brief This function is designed to be called once to perform all the checks needed
      * on the input provided. Checks can be "expensive" as the function is designed
      * to catch user's errors.
-     * @param r_model_part
+     * @param rModelPart The model part to compute
      * @return 0 all ok
      */
-    virtual int Check(ModelPart& r_model_part)
+    virtual int Check(ModelPart& rModelPart)
     {
         KRATOS_TRY
 
         return 0;
         KRATOS_CATCH("");
     }
 
-    //level of echo for the solving strategy
-    // 0 -> mute... no echo at all
-    // 1 -> printing time and basic informations
-    // 2 -> printing linear solver data
-    // 3 -> Print of debug informations:
-    //		Echo of stiffness matrix, Dx, b...
-
+    /**
+     * @brief It sets the level of echo for the solving strategy
+     * @param Level The level to set
+     * @details The different levels of echo are:
+     * - 0: Mute... no echo at all
+     * - 1: Printing time and basic informations
+     * - 2: Printing linear solver data
+     * - 3: Print of debug informations: Echo of stiffness matrix, Dx, b...
```

Comment:
there is also 4 - print to Matrix Market

Reply:
See next line